drydock/templates/kustomized/tutor13/base/kustomization.yml

```diff
@@ -10,7 +10,18 @@ resources:
 - k8s/jobs.yml
 {%- endif %}
 {%- if ENABLE_DRYDOCK_JOBS | default(false) %}
-- k8s/drydock-jobs.yml
+- k8s/jobs/mysql.yml
+- k8s/jobs/lms.yml
+- k8s/jobs/cms.yml
+{%- if RUN_MINIO | default(false) %}
```

Comment:
I think at the moment Drydock is adding a lot of variables and is hard to navigate them in the `config.yml` file. I think we can at least group them in a dict so it's a tiny bit easier to manage.